### ISSUE TITLE
bunch of JS errors after login

### ISSUE THREAD

**Issue by dmig** [pixel-saver/pixel-saver#149]
Running Ubuntu 17.10

No comments. Do you guys ever test your code?
```
Oct 26 22:40:24 dmig-tablet gnome-shell[23591]: JS WARNING: [/home/dmig/.local/share/gnome-shell/extensions/pixel-saver@deadalnix.me/decoration.js 212]: assignment to undeclared variable success
Oct 26 22:40:24 dmig-tablet gnome-shell[23591]: JS WARNING: [/home/dmig/.local/share/gnome-shell/extensions/pixel-saver@deadalnix.me/decoration.js 212]: assignment to undeclared variable pid
Oct 26 22:40:24 dmig-tablet gnome-shell[23591]: GNOME Shell started at Thu Oct 26 2017 22:40:21 GMT+0700
Oct 26 22:40:25 dmig-tablet gnome-shell[23591]: JS WARNING: [resource:///org/gnome/shell/ui/status/keyboard.js 708]: reference to undefined property "_inputSources"
Oct 26 22:40:29 dmig-tablet gnome-shell[23591]: JS ERROR: TypeError: windowList is null
                                                guessWindowXID@/home/dmig/.local/share/gnome-shell/extensions/pixel-saver@deadalnix.me/decoration.js:90:19
                                                onWindowAdded/<@/home/dmig/.local/share/gnome-shell/extensions/pixel-saver@deadalnix.me/decoration.js:282:12
```

**Comment by zapashcanon** [comment]
`JS ERROR: TypeError: windowList is null` should be fixed thanks to #101.

**Comment by tallero** [comment]
Oct 26 22:40:24 dmig-tablet gnome-shell[23591]: JS WARNING: 
    [/home/dmig/.local/share/gnome-shell/extensions/pixel-saver@deadalnix.me/decoration.js 212]: assignment to undeclared variable success
    Oct 26 22:40:24 dmig-tablet gnome-shell[23591]: JS WARNING: 
    [/home/dmig/.local/share/gnome-shell/extensions/pixel-saver@deadalnix.me/decoration.js 212]: assignment to undeclared variable pid

was corrected with #199.
